addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -65,7 +65,7 @@
                     string='Check Number'
                     attrs="{
                         'invisible': [('payment_method_code', '!=', 'new_third_party_checks'), '|', ('payment_method_code', '!=', 'check_printing'), ('l10n_latam_manual_checks', '=', False)],
-                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), ('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
+                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), '&amp;',('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
```

Comment:
```suggestion
                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), '&amp;', ('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
```

Reply:
@pablohmontenegro can you help me with this suggested change?
Thanks!

Reply:
@vbe-odoo yes! ready! Thanks!